SPECS/mariner-rpm-macros/gen-ld-script.sh

```diff
@@ -35,8 +32,8 @@ MODULE_INFO_DIR="$3/BUILD/"
 mkdir -pv $MODULE_INFO_DIR
 
 /usr/lib/rpm/mariner/generate-package-note.py \
-   --os "$OS_ID" \
-   --osVersion "$OS_VERSION" \
+   --os "mariner" \
+   --osVersion "2.0" \
```

Comment:
This might be hard to find whenever we switch versions. Why did we have to move away from parsing `/etc/os-release`?

Reply:
Wait, I see the reason in the changelog for this package. In that case maybe we could use the `mariner_release_version` macro (pass it as the 4<sup>th</sup> argument to this script)? We set it in our Go tooling, I think it should be available for all Mariner package builds and it relies on the value we already keep track of (I think it's the Mariner version set inside the main `Makefile`).

Reply:
Sounds good, I will change to pass mariner_release_version to the script